pkg/serial/handshake.go

```diff
@@ -32,10 +32,10 @@ import (
 )
 
 const (
-	SYN       = 0x16
-	ACK       = 0x06
-	DEBUG_ACK = 0x07
-	NAK       = 0x15
+	CodeSync     = 0x16
```

Comment:
What was the lint warning?  I'd rather not have `Code` in const name.  And `Syn` is probably fine.